apps/user/src/app/signup/page.tsx

```diff
@@ -40,6 +40,8 @@ const SignUpPage = () => {
                                 name="name"
                                 placeholder="이름을 입력해주세요."
                                 onChange={handleJoinUserDataChange}
+                                isError={joinUserData.name.length <= 0}
```

Comment:
```suggestion
                                isError={joinUserData.name.length == 0}
```
이렇게 고치는게 나아보여요. 왜냐하면 .length 프로퍼티의 값은 음수가 나올 수 없기도 하고 `글자의 수가 0보다 작다면` 보다 `글자의 수가 0이라면` 이라는 말이 덜 어색해보인다고 생각하는데 어떻게 생각하시나요?

Reply:
아 그러네요 제가 실수했네요 !! 동등연산자보다는 일치연산자가 더 좋은 코드가 될 것 같아요 ! 일치연산자 사용하여 수정하겠습니다